### ISSUE TITLE
Limited lateral neck flexion

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5942]
**Preferred term label:** Limited lateral neck flexion 

**Synonyms**

**Definition (free text, please give PubMed ID)** Reduced range of motion resulting in decreased ability to move the ear toward the top of the shoulder.

**Parent term (use hpo.jax.org/app)** Limited neck flexion HP:0005991

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term
